### DISCUSSION TITLE
Why is there freerdp2 and freerdp3?

### DISCUSSION THREAD

**Discussion by Utini2000** [FreeRDP/FreeRDP#10439] [Q&A]
Hi everyone, I just installed krdp on my home server to connect to it via the RDP client.

I noticed that I can't connect with freerdp3 (freerdp3 always wants a domain set and if none is entered, it will default to "ATHENA.MIT.EDU" which then ofcourse won't work!). 

I then figured out that krdp ships with freerdp2 on Arch as dependency and that doesn't require a domain to be set when connecting. So I am able to get it to work with freerdp2 while freerdp3 doesn't work but yet I wonder:

Why is there a freerdp2 and freerdp3 and not just one latest freerdp3?

Thanks!

**Comment by akallabeth** [comment]
1. no, `FreeRDP` does not default to `athena.mit.edu`, that sounds like the kerberos configuration of your system is broken. -> check https://github.com/freerdp/freerdp/wiki/FAQ
2. better ask questions about packaging in the arch forums/issue tracker/... we have nothing to do with that ;)

**Comment by Utini2000** [comment]
@akallabeth oh thanks for the hint regarding Kerberos. I never touched it but will try to figure out the issue.

Well freerdp.com still provides two packages / versions as well? It's 2.x and 3.x which both still get updated?

**Reply by akallabeth** [reply]
yes, we have stable (the 3.x versions) and oldstable (2.x versions)
as is the case with many programs when doing a new major version you need some transition time.
we´re using semantic versioning (see https://semver.org/ ) and `FreeRDP` is mostly a library used by other software (the `RDP` clients are more or less just a byproduct or reference implementation to show how it is used)

**Reply by Utini2000** [reply]
Thanks! Also commenting out all default values from arch "/etc/krb5.conf" fixed the domain issue!
